composer.lock

```diff
@@ -7742,6 +6915,5 @@
     "platform-dev": [],
     "platform-overrides": {
         "php": "7.2.9"
-    },
-    "plugin-api-version": "1.1.0"
```

Comment:
Looks like you use an older composer version than for the previous composer.lock change.

Reply:
Yes! I've upgraded my Composer to `1.10.6`, updated again and this was restored. Thanks for reviewing this!